content/post/wrong-data.md

```diff
@@ -0,0 +1,55 @@
+---
+author: Randy Clinton
+title: "Wrong data can still be right"
+publishDate: "????"
+cover: "????"
+draft: true
+tags: 
+ - data-integrity
+ - communication
+ - errors
+---
+
+Let us explore what it means for data to be wrong. For now, think of it from the user’s reaction only. It is a data point, trend line, or report that makes the user lean into their screen, squint, tilt their head a bit, and say, “nooooo, that’s wrong.”  
+
+Wanting to jump in and “fix” it is a natural feeling, especially for people who like the technical side of data. But let me make the case to pause for a beat and think through what “wrong” could mean before you head off fixing something that is not actually broken. 
+<!--more-->
+I propose there are three common situations in which data is considered wrong, and only one of them would fit a literal definition of “wrong.” They are (in order from least to most common):
+
+<ul>
+<li> The data consumer is using a faulty source to “check” the data.
+<li> The data is literally wrong. 
+<li> The data consumer misunderstands what the data represents. 
+</ul>
+
+## Begin with questions
+Recall the saying, “measure twice, cut once.” 
+
+Ask these two questions to the person who noticed the data being wrong (answer them yourself if you spotted the issue): What result would you expect to see here? Where would you check this number to know it’s right? A number missing a negative sign requires a solution miles away from a bug showing your sales a tenth of the actual number or off by a unit or two.
+
+Every time I look my old code I have new ideas for pretting it up, but when you’re on the hunt for the cause of wrong data keep an eye on the prize. Make time to deal with technical debt another time. 
+
+## Other data is wrong
+This is the least common of the three types of wrong, but common enough that it’s worth checking. One place you might see this is in web site statistics. Suppose you have an internal tracking system tracking site visits, and also use an external tracking tool. On your way to the office an email hits you -- “the site session count is wrong.” (_note: emails arriving between 5am and when you walk into the office are almost never good. No one gets an email saying “Hey we decided to give you a raise.”_). 
```

Comment:
Strike the trailing parenthetical here or move it into a footnote.

Reply:
done, not pushed.